src/mvs/workspace.h

```diff
@@ -42,12 +42,17 @@
 namespace colmap {
 namespace mvs {
 
+class ConfidenceMap;
+
 class Workspace {
```

Comment:
Maybe it's better to called to `NoCache` version `Workspace` and the cached one as `CachedWorkspace`.

Reply:
OK sounds good, I'll flip the inheritance structure around.